lib/Mojolicious/Command/prefork.pm

```diff
@@ -23,12 +23,21 @@ sub run {
     'k|keep-alive-timeout=i' => sub { $prefork->keep_alive_timeout($_[1]) },
     'l|listen=s'             => \my @listen,
     'P|pid-file=s'           => sub { $prefork->pid_file($_[1]) },
-    'p|proxy'                => sub { $prefork->reverse_proxy(1) },
+    'p|proxy:s'              => \my @proxy,
```

Comment:
Something like this would probably work too (with less code):
```
'p|proxy:s' => sub { $_[1] eq '' ? $daemon->reverse_proxy(1) : push @{$daemon->trusted_proxies}, $_[1] },
```

Reply:
this ends up being too long